debian/pop-default-settings.postinst

```diff
@@ -99,8 +107,22 @@ case "$1" in
             echo "${ISO_KEY}" | apt-key add -
             echo "${SOURCE}" > /etc/apt/sources.list.d/pop-os-apps.sources
         fi
-        ;;
 
+        # Convert system sources from one-line format to DEB822, if present
+        # Generous matching, be more specific if found
+        if grep -e "ubuntu.com" -e "apt.pop-os.org/ubuntu" /etc/apt/sources.list 
```

Comment:
You can add /ubuntu to all the ubuntu.com lines just like on apt.pop-os.org/ubuntu

Reply:
Updated.